### DISCUSSION TITLE
Add Tor support through Orbot

### DISCUSSION THREAD

**Discussion by ghost** [Docile-Alligator/Infinity-For-Reddit#895] [Ideas]
RedReader has this feature where all connections go through Orbot to give maximum Anonymity. I would love to have that as a feature.

_Also I want to shill #399 for a second. I was going to suggest full Anon mode that would include local subscription, saved post support and the like but found out there's already an issue._

**Comment by whereismybugfix** [comment]
I would vote for that feature too. It might be done via SOCKS5 proxy support, pretty common feature among the apps that have Internet access.

**Comment by ghost** [comment]
I would like to vouch for this feature again, this and proper offline subscriptions and saves are the only things that keep me from staying anonymous.

**Reply by JayKx** [reply]
Absolutely necessary feature for privacy. When enabled app shouldn't connect to reddit.

**Comment by onkq** [comment]
Note: even though it originally came from an acronym, Tor is not spelled "TOR". Only the first letter is capitalized. In fact, we can usually spot people who haven't read any of our website (and have instead learned everything they know about Tor from news articles) by the fact that they spell it wrong.
https://support.torproject.org/about/why-is-it-called-tor/
ffs people

**Reply by ghost** [reply]
It's semantics but I fixed it.
